advanced/Templates/gravity.db.sql

```diff
@@ -48,11 +83,12 @@ INSERT INTO info VALUES("version","1");
 
 CREATE VIEW vw_gravity AS SELECT a.domain
     FROM gravity a
-    WHERE a.domain NOT IN (SELECT domain from whitelist WHERE enabled == 1);
+    WHERE a.domain NOT IN (SELECT domain from vw_whitelist);
 
 CREATE VIEW vw_whitelist AS SELECT a.domain
     FROM whitelist a
-    WHERE a.enabled == 1
+    INNER JOIN domain_groups b ON b.id = a.group_id
```

Comment:
`INNER` is not necessary here, as it is the default.

Reply:
My SQL book recommends to explicitly write `INNER JOIN`.

> You might see queries where the `INNER JOIN` is written simply as a `JOIN`. These two are equivalent, but we will continue to refer to these joins as inner-joins because they make the query easier to read once you start using other types of joins, which will be introduced in the following lesson.